src/OpenSage.Game/Logic/Object/Update/AIUpdate/HackInternetAIUpdate.cs

```diff
@@ -97,6 +91,12 @@ public void StartHackingInternet()
                 throw new InvalidStateException();
             }
 
+            GameObject.ModelConditionFlags.Set(ModelConditionFlag.Unpacking, true);
+            GameObject.ModelConditionFlags.Set(ModelConditionFlag.FiringA, false);
+            GameObject.ModelConditionFlags.Set(ModelConditionFlag.Packing, false);
```

Comment:
Since `Update` isn't called until later, I had to move this to here, so we can then update the animation duration below.